frontend/admin/src/js/components/user/useFilterOptions.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  getWorkRegion,
+  getEmploymentDuration,
+  getLanguageAbility,
+  getOperationalRequirement,
+  getEducationType,
+  getJobLookingStatus,
+} from "@common/constants/localizedConstants";
+import { enumToOptions } from "@common/helpers/formUtils";
+import mapValues from "lodash/mapValues";
+import { useIntl } from "react-intl";
+import useLocale from "../useLocale";
+import {
+  WorkRegion,
+  OperationalRequirement,
+  EducationType,
+  JobLookingStatus,
+  LanguageAbility,
+  useAllSkillsQuery,
+  useGetClassificationsQuery,
+  useGetPoolsQuery,
+} from "../../api/generated";
+
+// This is use way to remove null and undefined values from list types, which
+// makes working with them simpler (e.g. `arr.map( ... )`)
+// Source: https://stackoverflow.com/a/46700791
+function notNullOrUndefined<TValue>(
```

Comment:
I think this is a duplicate of already existing `notEmpty` in `frontend/common/src/helpers/util.ts`

Reply:
Good call, will pull that. Thanks!